scripts/publish-ghpages.sh

```diff
@@ -0,0 +1,27 @@
+#/bin/sh
+
+set -o errexit -o nounset
+
+# User info
+git config user.name "patternfly-build"
+git config user.email "patternfly-build@redhat.com"
+git config --global push.default simple
+
+## Deploy to github pages
+if [ -d "patternfly.github.io" ]; then
+  rm -rf patternfly.github.io
+fi
+git clone git@github.com:patternfly/patternfly.github.io.git
+rsync -av --delete --exclude .git source/_site/ patternfly.github.io
+cd patternfly.github.io
+find components -type f -not -regex ".*/.*\.\(html\|js\|css\|less\)" -print0 | xargs -0 rm
```

Comment:
All non web resources.  The bower components brings in things like exe binaries etc.  This is a hack.  When we move to consuming the git submodules and npm dependencies instead of a blind bower components copy this can be removed.